lab1b/lexic.txt

```diff
@@ -0,0 +1,32 @@
+Alphabet:
+    a. A-Z, a-z
+    b. decimal digits
+    c. underscore
+
+1. Lexic:
+
+    I. special symbols
+        a. operators: + - * / % < <= = == >= > !=
+        b. separators: () [] {} , ; space
+        c. reserved words:
+            - const, number, string
+            - print, for, while, in, read_integer, read_string
+    
+    II. identifiers
+        <digit> ::= 0|1|2|3|4|5|6|7|8|9
+        <nonzero_digit> ::= 0|1|2|3|4|5|6|7|8|9
+        <letter> ::= a|b|...|z|A|B|...|Z
+        <special_symbol> ::= _|$|+|-|*|/|%|<|=|>|!|space
```

Comment:
do you really want space to be considered a special symbol? It will greatly complicate the parsing. Just a heads up